test/forms/CancelDeclarationSpec.scala

```diff
@@ -17,105 +17,110 @@
 package forms
 
 import base.UnitSpec
+import play.api.data.Form
 import play.api.libs.json.{JsObject, JsString}
 
 class CancelDeclarationSpec extends UnitSpec {
 
-  def formData(lrn: String = "lrn", mrn: String = "123456789012345678", description: String = "description", reason: String = "1") =
-    JsObject(
-      Map(
-        "functionalReferenceId" -> JsString(lrn),
-        "mrn" -> JsString(mrn),
-        "statementDescription" -> JsString(description),
-        "changeReason" -> JsString(reason)
-      )
+  private val validMrn = "123456789012345678"
+  def getBoundedForm(lrn: String = "lrn", mrn: String = validMrn, description: String = "description", reason: String = "1") =
+    CancelDeclaration.form.bind(
+      JsObject(
```

Comment:
`bind(JsValue)` is deprecated. You can directly use here `bind(Map)`. No need to convert the map to json.

Reply:
Not the version I used (that includes the `MaxChars` param) :slightly_smiling_face:

Reply:
OK, but when you already have a Map when not directly use it? Why the need to convert it to json?